doc/release_notes/release_notes_1.2.rst

```diff
@@ -0,0 +1,352 @@
+* :ref:`Supported Hardware <hardware>`
+* :ref:`Using Hybrid mode on NUC <using_hybrid_mode_on_nuc>`
+* :ref:`Launch Two User VMs on NUC using SDC2 Scenario <using_sdc2_mode_on_nuc>`
+
+New Features Details
+********************
+
+- :acrn-issue:`3051` - Kata Container Storage: Support rescan feature for virtio-blk
+- :acrn-issue:`3163` - WaaG: USB host support
+- :acrn-issue:`3401` - System is lagging when execute "fdisk -l" to query disk information with multiple USB devices after launch or reboot UOS
+- :acrn-issue:`3486` - USB mediator: mediator mode can not recognize the SSD disk in RTVM and LaaG
+- :acrn-issue:`3501` - Virtualization supports Always Running Timer (ART)
+- :acrn-issue:`3505` - Succeed to reboot the whole system while execute reboot command in SOS
+- :acrn-issue:`3506` - OVMF release v1.2
+
+Fixed Issues Details
+********************
+
+- :acrn-issue:`2326` - There is no Read value output in the log when test "ST_PERF_USB2/3_mass_storage_protocol_Read_Large_File_as_Host"
+- :acrn-issue:`2927` - The android guest will hung after test USB mediator camera[Frequency:20%]
+- :acrn-issue:`3027` - Reboot SOS Failed[Frequency:10%]
+- :acrn-issue:`3152` - Use virtio-blk instead passthru devices to boot RT
+- :acrn-issue:`3181` - [auto][sit][daily]Case "Hypervisor_Launch_RTVM_on_SATA_Storage" sata disk can not passthru
+- :acrn-issue:`3239` - HV can not produce #GP correctly sometimes
+- :acrn-issue:`3268` - dm: add virtio-rnd device to command line
+- :acrn-issue:`3277` - Potential Memory Leaks Found
+- :acrn-issue:`3279` - AcrnGT causes display flicker in some situations
+- :acrn-issue:`3280` - AcrnGT holding forcewake lock cause high cpu usage gvt workload thread
+- :acrn-issue:`3281` - AcrnGT emulation thread causes high cpu usage when shadowing ppgtt
+- :acrn-issue:`3283` - New scenario-based configurations lack documentation
+- :acrn-issue:`3341` - Documentation on how to run Windows as a Guest (WaaG)
+- :acrn-issue:`3370` - vm_console 2 cannot switch to VM2’s console in hybrid mode
+- :acrn-issue:`3374` - Potential interrupt info overwrite in acrn_handle_pending_request
+- :acrn-issue:`3379` - DM: Increase hugetlbfs MAX_PATH_LEN from 128 to 256
+- :acrn-issue:`3392` - During run UnigenHeaven 3D gfx benchmark in WaaG, RTVM lantency is much long
+- :acrn-issue:`3466` - Buffer overflow will happen in 'strncmp' when 'n_arg' is 0
+- :acrn-issue:`3467` - Potential risk in virtioi_i2c.c & virtio_console.c
+- :acrn-issue:`3469` - [APL NUC] Display goes black while booting; when only one display monitor is connected
+- :acrn-issue:`3473` - dm: bugfix for remote launch guest issue
+- :acrn-issue:`3480` - Add script to ``acrn-config/target`` and ``acrn-config/host/board_confing``
+- :acrn-issue:`3482` - Acrn-hypervisor Root Directory Clean-up and Create misc/ folder for Acrn daemons, services and tools
+- :acrn-issue:`3486` - USB mediator: mediator mode can not recognize the SSD disk in RTVM and LaaG
```

Comment:
I just noticed 3486 is in both Fixed Issues and New Features too.

Reply:
Remove it from the New Feature list, sorry for the mistake.